### DISCUSSION TITLE
Having multiple Zeit accounts for each of your clients/projects

### DISCUSSION THREAD

**Discussion by babyPrince** [vercel/vercel#4058] [General] [answered]
Hi Freelancers,
How are you guys handling different Zeit accounts for each of your Clients/Projects (for separate billing plans)? Seems I have to have a new Github account for each client registration on Zeit - that might be madness to keep switching between locally.

Curious why email registration is not available?

**Comment by coetry** [comment]
@babyPrince Why not put all of your client projects under a single agency team account? We intentionally don't support only email sign-ups since our platform is catered for developers and this also acts as abuse prevention.

**Comment by babyPrince** [comment, marked as answer]
@coetry By single agency team account you mean Pro account on Zeit and add every client as Team, or Team/Organisations on github? It's a bit confusing

If the first, does it mean that for each team (client/project) it will cost $20 per month just for me as developer to have access?

**Reply by mcsdevv** [reply]
If your projects are all in the same Git account, you can use a single team.

If your projects are in multiple Git accounts, you'll need to create a team for each which would cost $20 a month per team. If that doesn't make economical sense at the moment, it may well be worth looking at ways to bring them under a single Git account.

**Reply by babyPrince** [reply]
And if they are under single team and single git account, is the zeit usage (builds, serv.functions, etc) shared between each project?

**Reply by mcsdevv** [reply]
That's correct, all usage is at the account level and shared between all projects.
